po/pl.po

```diff
@@ -1676,6 +1757,9 @@ msgstr "Ładowanie sampli"
 msgid "Sample Size:"
 msgstr "Rozmiar sampli:"
 
+msgid "Sample Rate:"
+msgstr "Częstotliwość sampli:"
```

Comment:
```suggestion
msgstr "Częstotliwość próbkowania:"
```

Reply:
Now sometimes 'sampli', sometimes 'probki' are used in the ui. May be we choose one of them?

Reply:
That's due to another language curiosity: when talking about samples as in sample sets, the imported word (sample, sampel, sampli, etc.) is often used. When signal processing is discussed (what I believe is what this "sample rate" describes), the term "próbkowanie" is used almost exclusively. While "próbka" and its variants could be used (and be understood) in the other contexts (próbki organów, sample organów both seem OK), when talking about signal processing "częstotliwość sampli" sounds unnatural.

Reply:
But "Sample rate" and "Samples per buffer" are neighbours at the "Audio" page. Using different words for them would look strange.

Reply:
Indeed, those all seem to be related to the signal processing part and probably should be changed as well.

Furthermore, from what I can see, "próbki" is also used for sample sets in many places. Probably the best way would be to use "próbki" universally and just drop the loanword which may sound strange in some contexts anyway.

Reply:
OK. I've replaced both with ``Częstotliwość próbkowania`` and ``Próbek na bofor``